### ISSUE TITLE
Unknown Command

### ISSUE THREAD

**Issue by spot88** [BatMen/csgo-retakes-splewis-convar-weapon-allocator#14]
Hello, 

Unknown Command:
sm_retakes_weapon_nades_hegrenade_ct_enabled 0
sm_retakes_weapon_nades_hegrenade_t_enabled 0
sm_retakes_weapon_nades_flashbang_ct_enabled 1
sm_retakes_weapon_nades_flashbang_t_enabled 1
sm_retakes_weapon_nades_smokegrenade_ct_enabled 0
sm_retakes_weapon_nades_smokegrenade_t_enabled 0
sm_retakes_weapon_nades_molotov_ct_enabled 0
sm_retakes_weapon_nades_molotov_t_enabled 0
sm_retakes_weapon_awp_enabled 1
sm_retakes_weapon_gunrounds 5

See picture, plugins is loaded as normal. 
![plugins-list](https://cloud.githubusercontent.com/assets/10884072/12112862/8ccaad34-b39f-11e5-86af-ea8be6352840.png)

Has there been a change on sm commands?
sm_ commands are taken from your github Explanations.

**Comment by BatMen** [comment]
Yes, Commands for nades changed from "_enabled" to "_max". README.md is up to date.

For example : sm_retakes_weapon_nades_hegrenade_ct_max 3 if you want 3 hegrenades max for ct team.

**Comment by spot88** [comment]
What about the awp_enable and gunarounds?

**Comment by BatMen** [comment]
No this is a mistake in README.md 

I have to change it.
